### PR TITLE
add documentation about custom formatter in readme

### PR DESCRIPTION
**Pull request by avadan1**
Add documentation in README about [most recent feature](https://github.com/vendasta/yeti_logger/pull/9).

@vendasta/meerkats 
@vendasta/sre

### CODE REVIEW

**Review by lmelnyk-va — APPROVED** [review]

**Comment by mvegac** [comment]
I suggest waiting for at least one review from the Yesware team before merging. But LGTM.

**Comment by avadan1** [comment]
> I suggest waiting for at least one review from the Yesware team before merging. But LGTM.

Sorry for jumping the gun. I'm open to suggestions to change! Will open a new PR.

**Pull request merged**
